.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+/logs/
```

Comment:
Consider using a consistent pattern for ignoring directories by adding a trailing slash to the 'logs' entry, like 'logs/'. This ensures that the directory is ignored, not just a file named 'logs'.